cirq-google/cirq_google/engine/virtual_engine_factory.py

```diff
@@ -367,3 +370,36 @@ def create_noiseless_virtual_engine_from_latest_templates() -> SimulatedLocalEng
     processor_ids = list(MOST_RECENT_TEMPLATES.keys())
     template_names = [MOST_RECENT_TEMPLATES[k] for k in processor_ids]
     return create_noiseless_virtual_engine_from_templates(processor_ids, template_names)
+
+
+def create_default_noisy_quantum_virtual_machine(
+    processor_id: str, simulator_class: Type[SimulatesSamples], **kwargs
```

Comment:
is awkward that there isn't a default for simulator_class. I'd strongly consider having a default here.

I assume this is so you can inject a QSim simulator because performance is likely important when dealing with noise models (?)

honestly I would probably advocate for some python trickery where you attempt a local import of qsimcirq and fall back to cirq simulator


```python
def create_xxx(processor_id: str, simulator_class = None):
  if simulator_class is None:
    try:
      import qsimcirq
      simulator_class = qsimcirq.Simulator
    except ImportError:
      simulator_class = cirq.Simulator
  ...
```

Reply:
I like the default trickery. Added